kapitel/kapitel1.tex

```diff
@@ -0,0 +1,54 @@
+
+%Versand
+Eine weitere Funktion ist die des Dokumentenversands. Bei Drücken dieses Knopfs öffnet sich ein Modal wo der Nutzer die gewünschten Dokumente aus allen relevanten Dokumenten auswählen kann. Dazu muss noch eine, oder mehrere E-Mails angegeben werden und dann werden die Dokumente per E-Mail in einem ZIP-Archiv versendet. Die Dokumente beinhalten rechtliche Informationen und der Antrag, sowie das Angebot können auch ausgewählt werden, mit der gleichen Einschränkung wie bei der Druckfunktionalität beschrieben. Die komplette Dokumentenversandkomponente gibt es als eigenständige Bibliothek, da sie auch für andere \ac{OTR} genutzt wird, so dass diese für Frontend nur eingebunden werden musste.
+
+\subsection{Projektbegründung}
+\label{projektbegründung}
+Die Motivation hinter diesem Projekt begründet sich in der einfachen Erweiterbarkeit für neue Versicherungsprodukte, so wie eine Überholung des Aussehens des \ac{OTR}. Allgemein ist der \ac{OTR} durch den modularen und komponentenbasierten Aufbau des Projekts besser wartbar und Einzelteile einfacher auszutauschen. \\
+Die Fußzeile ist eine Komponente die in allen bestehenden Produkten und zukünftig kommenden Produkten vorhanden sein muss. Die Position bietet sich außerdem für die gegeben Funktionalitäten gut an, da sie unabhängig von dem Kontext der Rest der Webseite funktionieren soll.
+\subsection{Projektschnittstellen}
+\label{projektschnittstellen}
+Das Backend hat %, wie schon erwähnt,
+Schnittstellen zu einigen Systemen. Es gibt das \ac{TG}, welches den Einstiegspunkt von Extern zu den Tarifrechnern darstellt. Dieses, sowie der \ac{OTR} benutzt das \ac{TRG}, welches als REST-Schnittstelle funktioniert um unter anderem Konfigurationen für Tarife oder das Speichern von Angeboten anzubieten. Das \ac{TRG} speichert Daten in eine MS SQLServer Datenbank, während der neue \ac{OTR} keine direkte Anbindung an eine Datenbank hat. \\
+Weitere Schnittstellen und Systeme die von der Fußzeilenkomponente genutzt werden ist der Mailservice, eine Anwendung welche E-Mails verschickt, sowie der PDFToolsservice, welcher die Generierung von PDF-Dokumenten übernimmt. Die Fußzeile, so wie alle Komponenten im Frontend bekommen ihre Daten über den Vue.js Pinia Store geliefert.
```

Comment:
- MailService würde ich nicht beschreiben, hast du vorher schon mit der fertigen Dokumentenversand-Komponente quasi gemacht. Und das ist einer Fertig Komponente, so wie du es geschrieben hast
- "Vue.js Pinia Store" in Klammern schreiben und dafür lieber Localstorage des Browser